src/test/groovy/hudson/plugins/gradle/injection/GradleSnippets.groovy

```diff
@@ -0,0 +1,51 @@
+package hudson.plugins.gradle.injection
+
+import hudson.plugins.gradle.GradleInstallationRule
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition
+import org.jenkinsci.plugins.workflow.job.WorkflowJob
+import org.jvnet.hudson.test.JenkinsRule
+
+class GradleSnippets {
+
+    def static pipelineJobWithError(JenkinsRule j, GradleInstallationRule gradleInstallationRule) {
```

Comment:
I think it's better to use a concrete type instead of `def`. It's not easy to understand what's being returned here